recipes/openvdb/all/conandata.yml

```
@@ -4,5 +4,9 @@ sources:
     sha256: "a6845da7c604d2c72e4141c898930ac8a2375521e535f696c2cd92bebbe43c4f"
 patches:
   "8.0.1":
-    - patch_file: "patches/0001-Find-packages-fixes.patch"
-      base_path: "source_subfolder"
+    - patch_file: "patches/FindIlmBase.patch"
+      patch_description: "Update FindIlmBase.cmake"
+      patch_type: "backport"
+    - patch_file: "patches/FindTBB.patch"
+      patch_description: "Update FindTBB.cmake"
+      patch_type: "backport"
```

```suggestion
      patch_type: "conan"
```
nothing has been added, only the target is fixed.